harvard-stellenbosch-university.csl

```diff
@@ -203,7 +203,7 @@
     <layout prefix="(" suffix=")" delimiter="; ">
       <group delimiter=", ">
         <text macro="author-short"/>
-        <text macro="date-issued-no-parentheses"/>
+        <text macro="year-date"/>
```

Comment:
You've now called a macro (year-date) that does not exist and therefor it is giving you an error.
What are you trying to accomplish?